ert_shared/plugins/hook_specifications/logging.py

```diff
@@ -2,11 +2,9 @@
 
 
 @hook_specification
-def add_log_handle(logging):
+def add_log_handle_to_root():
     """
-    Modify ert's logging instance, for example adding handlers and changing formatting.
+    Adding handlers to the root logger in ert.
```

Comment:
Propsed change to docstring to hopefully make it more clear to those implementing hooks what we expect.

Return a list of one or multiple log handlers that you want added to ERT's root-logger.

Reply:
Agree that we can improve the doc string, but we can not return a list, you would have to add a single handler per plugin. I think that is ok, but that is a change we make intentionally ;)

Reply:
Ah I see, so something like "Return a log handler that you want added to ERT's root-logger"?